yaml-tests/yaml-tests.gradle

```diff
@@ -128,8 +128,16 @@ task downloadExternalServer(type: Copy) {
     into project.layout.buildDirectory.dir('externalServer')
 }
 
+task buildLocalServer(type: Copy) {
+    dependsOn ":fdb-relational-server:package", "downloadExternalServer"
+    from rootProject.project("fdb-relational-server").layout.files('.dist')
+    include("**/*-all.jar")
```

Comment:
Should this be more specific instead of `*-all.jar`?

Reply:
Changed to be tighter.